File: src/mod1_syntax/syntax/lesson19_functions/FunctionsDemo.java
package mod1_syntax.syntax.lesson19_functions;

public class FunctionsDemo {
    public static void main(String[] args) {

        System.out.printf("Сумма 15 и 35 равна: %d%n", sumOfNumber(15, 35));

    }

    public static long sumOfNumber(int a, int b) {
        return a + b;
    }
}

File: src/mod1_syntax/syntax/lesson19_functions/FunctionsDemo.kt
package mod1_syntax.syntax.lesson19_functions

fun main() {

    println("Получаем сумму чисел от 0 до 10: ${getSumOfNumbers(numberRange = 10)}")
    println("Получаем сумму двух чисел 4 и 8: ${getSumOfTwoNumbers(4, 8)}")

}

fun getSumOfNumbers(numberRange: Int): Int {
    var cnt = 0
    for (number in 0..numberRange) {
        cnt += number
    }
    return cnt
}

fun getSumOfTwoNumbers(a: Int, b: Int): Long = (a+b).toLong()